File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelNests/IfcRelNests.vb
Public Class IfcRelNests Inherits IfcRelDecomposes
    Public RelatingObject As IfcObjectDefinition
    Public RelatedObjects As List(Of IfcObjectDefinition)

    ' === WHERE CLAUSES ===
    ' NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelNests/IfcRelNests.cs
public class IfcRelNests : IfcRelDecomposes
{
    public IfcObjectDefinition RelatingObject { get; set; }
    public List<IfcObjectDefinition> RelatedObjects { get; set; }

    // === WHERE CLAUSES ===
    // NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
}
